src/beamlime/core/processor.py

```diff
@@ -32,12 +32,12 @@ def __init__(
         logger: logging.Logger | None = None,
         source: MessageSource[Tin],
         sink: MessageSink[Tout],
-        handler_factory: HandlerFactory[Tin, Tout],
+        handler_registry: HandlerRegistry[Tin, Tout],
```

Comment:
We now have to pass the registry, since we want to be able to insert a custom handler for config messages, i.e., the processor cannot create the registry itself.